tools/mo/requirements_onnx.txt

```diff
@@ -1,7 +1,7 @@
 onnx>=1.8.1,<=1.12
 networkx~=2.5; python_version <= "3.6"
 networkx<=2.8.8; python_version > "3.6"
-numpy>=1.16.6,<=1.24.0
+numpy>=1.16.6,<1.25.0
```

Comment:
```suggestion
numpy>=1.16.6,<1.24
```
Am I wrong?
> Python bindings numpy version stays on 1.23 beacuse ONNX==1.13 does not support numpy==1.24.

Reply:
I'm not certain - ONNX doesn't support numpy==1.24 only when it comes to running ONNX backend tests through OV, since MO is a separate component I think it technically *could* be bumped. We could downgrade the numpy version just to be safe though.

Reply:
@p-wysocki, can you check please if MO works for ONNX models converted by the legacy MO `--use_legacy_frontend` using your requirements?

Reply:
No, I didn't find them. I looked for models for which we fallback to the legacy ONNX.

Reply:
@rkazants I converted `yolov3-tiny.onnx` using master version of MO and `numpy==1.24`, the IR has been generated successfully. Is this test sufficient?

Reply:
yes, it is. Thanks!